app/assess/templates/macros/banner_summary.html

```diff
@@ -1,4 +1,4 @@
-{% macro banner_summary(fund_name, project_reference, project_name, funding_amount_requested, workflow_status) %}
+{% macro banner_summary(fund_name, project_reference, project_name, funding_amount_requested, workflow_status, flag=None) %}
```

Comment:
Did you also add a change somewhere that calls `banner_summary` with a flag? I can't see it in this PR